easy-ui-react/src/Notification/Notification.tsx

```diff
@@ -357,7 +357,6 @@ export function NotificationProvider(props: NotificationProviderProps) {
   const state = {
     close: combinedState.close,
     add: combinedState.add,
-    remove: combinedState.remove,
```

Comment:
React Aria removed this from their `ToastState`. didn't see this used anywhere so it seems safe